internal/pkg/agent/cmd/upgrade.go

```diff
@@ -31,15 +32,19 @@ const (
 	flagPGPBytes       = "pgp"
 	flagPGPBytesPath   = "pgp-path"
 	flagPGPBytesURI    = "pgp-uri"
+	flagForce          = "force"
 )
 
+var unsupportedUpgrade error = errors.New("upgrading fleet managed agents is not supported")
```

Comment:
```suggestion
var unsupportedUpgrade error = errors.New("this agent is Fleet managed and must be upgraded using Fleet")
```

Upgrading fleet managed agents is supported, just from fleet. Also let's avoid the word "unsupported" since there may be cases where we explicitly tell users to do this because of bugs and we don't want to think they are doing something fundamentally risky.